coalib/settings/Section.py

```diff
@@ -43,6 +46,53 @@ def append_to_sections(sections,
         key, str(value), origin, from_cli=from_cli, to_append=to_append))
 
 
+def extract_aspects_from_section(section):
+    """
+    Extract aspects and its related setting from a section and create an
+    aspectlist from it.
+
+    :section: Section object.
+    :return:  aspectlist containing aspectclass instance with user-defined
+              tastes.
+    """
+    if not len(section.get('aspects')):
+        # Off switch for aspect-based configuration
```

Comment:
this comment is not clear and simple.
And `None` should be described in the docstring.

And add a blank line in the code after this `if` , like you have done after the `raise` below.